src/test/java/core/basesyntax/operations/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static PurchaseOperationHandler operationHandler;

```

Comment:
```suggestion
    private static OperationHandler operationHandler;
```
the same for other implementations test classes